src/sql/run/sparkdataframe.py

```diff
@@ -1,19 +1,27 @@
 try:
     from pyspark.sql import DataFrame
     from pyspark.sql.connect.dataframe import DataFrame as CDataFrame
+    from pyspark.sql.utils import AnalysisException
 except ModuleNotFoundError:
     DataFrame = None
     CDataFrame = None
+    AnalysisException = None
 
 from sql import exceptions
 
 
 def handle_spark_dataframe(dataframe, should_cache=False):
     """Execute a ResultSet sqlaproxy using pysark module."""
-    if not DataFrame and not CDataFrame:
+    if not DataFrame and not CDataFrame and not AnalysisException:
         raise exceptions.MissingPackageError("pysark not installed")
 
-    return SparkResultProxy(dataframe, dataframe.columns, should_cache)
+    try:
```

Comment:
please integrate this with short_errors: https://github.com/ploomber/jupysql/blob/043344400de468f5d340104c65ec78be5a217c7f/src/sql/magic.py#L175

by default, it should raise the exception, if short_errors is True, then just print it

Reply:
I see this was marked as resolved but I don't see any references to the `short_errors` option in your PR, am I missing anything?